saleor/channel/migrations/0007_order_settings_per_channel.py

```diff
@@ -7,14 +7,20 @@ def set_order_settings(apps, schema_editor):
 
     site_settings = SiteSettings.objects.first()
 
-    Channel.objects.update(
-        automatically_confirm_all_new_orders=(
-            site_settings.automatically_confirm_all_new_orders
-        ),
-        automatically_fulfill_non_shippable_gift_card=(
-            site_settings.automatically_fulfill_non_shippable_gift_card
-        ),
-    )
+    if site_settings:
```

Comment:
You could limit the `.update` call to single entry (but this is only a cosmetic change. Don't know if you like it)
```
automatically_confirm_all_new_orders=True
automatically_fulfill_non_shippable_gift_card=True
if site_settings:
    automatically_confirm_all_new_orders=site_settings.automatically_confirm_all_new_orders
    automatically_fulfill_non_shippable_gift_card=site_settings.automatically_fulfill_non_shippable_gift_card
Channel.objects.update(
             automatically_confirm_all_new_orders= automatically_confirm_all_new_orders,
             automatically_fulfill_non_shippable_gift_card= automatically_fulfill_non_shippable_gift_card,
)

Reply:
Looks good, will change that.